models.py

```diff
@@ -47,3 +47,25 @@ class Organisation(BaseModel):
     nombre_repertoires: int
     date_creation: str
     plateforme: str
+
+@dataclass(frozen=True)
+class Package(BaseModel):
+    deprecation_reason: bool
```

Comment:
I haven't closely checked about deprecation_reason being a bool, I assume you did, please ignore this if so.

Reply:
I have not yet actually. I took this schema as a dumb dev :)